css/responsive/mobile.styl

```diff
@@ -100,4 +100,11 @@ $margin-large = ($gutter * 5)
     height: 50vh
     padding: $margin-small
     &:nth-child(1)
-      border-bottom: $border
\ No newline at end of file
+      border-bottom: $border
+
+// ******************************************************
+
+// HOME
+
+.home-item
+  padding: $margin-small
```

Comment:
<img width="270" alt="screen shot 2016-08-24 at 5 36 41 pm" src="https://cloud.githubusercontent.com/assets/7727722/17952645/5c6c150e-6a21-11e6-828b-98df359149a5.png">
aligns with the container on mobile